plugins/woocommerce-admin/client/payments-welcome/index.tsx

```diff
@@ -52,7 +52,7 @@ interface activatePromoResponse {
 	success: boolean;
 }
 
-const PROMO_NAME = 'wcpay-promo-2022-3-incentive-100-off';
+const PROMO_NAME = 'wcpay-promo-2022-us-incentive-25-off';
```

Comment:
The text copies say "20% off", but this promo name implies it's 25% off. Is this correct?

Reply:
Good catch! We modified the promotion after updating the ID and I forgot to update it again. Addressed in 61a7a35.